sqldelight-gradle-plugin/src/test/fixtures/keyword_identifiers/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test (
+  "DESC" TEXT,
```

Comment:
nvm the SQLite author apparently wants you to quote every single english word you use in case it gets keyword'd in a future version........lol